eng/Versions.props

```diff
@@ -198,10 +196,6 @@
       framework in current .NET SDKs.
     -->
     <MicrosoftNETTestSdkVersion>17.1.0-preview-20211109-03</MicrosoftNETTestSdkVersion>
-    <!--
-      Use a compiler new enough to use required keyword
-    -->
-    <MicrosoftNetCompilersToolsetVersion>4.4.0-1.22358.14</MicrosoftNetCompilersToolsetVersion>
```

Comment:
Arcade version of this version property is currently `4.3.0-3.22365.10`, meaning relying on the .NET SDK makes sense.